### ISSUE TITLE
Medusa 1.1 package can't be upgraded as backup can't be done

### ISSUE THREAD

**Issue by Teryon** [BenjV/SYNO-packages#20]
I'm using your Medusa python3 package for some weeks now, without any problems but I just found out that Medusa can't be updated becausse it start a backup that's failling.

So how can I correct this ?

**Comment by BenjV** [comment]
Maybe a rights issue.
Make sure the group sc-download have write access to the location you want to put the backup

**Comment by Teryon** [comment]
Hi,
Manual backup let me choose where I want to backup is working great.
Update process start a backup whitout letting me choose where to backup it and so backup fails and update doesn't happen.

**Comment by BenjV** [comment]
Strange.
Any Idea where that update process wants to place that backup?
I would expect it on the "datadir" (e.g. /var/packages/medusa/target/var) and that should not be a problem.

**Comment by Teryon** [comment]
It's strange !
Logs : 
2020-11-14 18:18:57 INFO     APIv2-Thread_15 :: [552f119] Checking for updates using GitHub Updater
2020-11-14 18:19:04 INFO     APIv2-Thread_3 :: [552f119] Config backup in progress...
2020-11-14 18:20:07 INFO     APIv2-Thread_3 :: [552f119] Config backup successful

But I ended up with a red cross, and nothing next.

I tried from an another computer and it updated withtout any hick-ups.
The only differences was I use my local IP adress instead of my domain name (throught a reverse proxy.

Whatever, thank you for your help and sorry for the trouble.
